.zenodo.json

```diff
@@ -0,0 +1,51 @@
+{
+  "creators": [
+    {
+      "name": "Ehbrecht, Carsten",
+      "affiliation": "DKRZ"
+    },
+    {
+      "name": "Charette-Migneault, Francis",
+      "affiliation": "CRIM",
+      "orcid": "0000-0003-4862-3349"
+    },
+    {
+      "name": "Derue, François-Xavier",
+      "affiliation": "CRIM"
+    },
+    {
+      "name": "Smith, Trevor James",
+      "affiliation": "Ouranos",
+      "orcid": "0000-0001-5393-8359"
+    },
+    {
+      "name": "Byrns, David",
+      "affiliation": "Intact"
+    },
+    {
+      "name": "Perron, Louis-David",
+      "affiliation": "CRIM"
+    }
+  ],
+  "keywords": [
+    "security",
+    "ows",
+    "PAVICS",
+    "Weaver",
+    "Magpie",
+    "bird-house",
+    "wps",
+    "web processing services"
```

Comment:
Somewhat duplicate.

Also, although the code has more specific considerations for WPS, it can do any OWS. 
So maybe add `OGC Web Services` in there also.